cms/models/pagemodel.py

```diff
@@ -179,7 +181,7 @@ def move_page(self, target, position='first-child'):
                 if target.get_root().get_next_sibling().pk == public.get_root().pk:
                     target = target.publisher_public
                 else:
-                    Logger.warning('tree may need rebuilding: run manage.py cms fix-tree')
+                    logger.warning('tree may need rebuilding: run manage.py cms fix-tree')
```

Comment:
This is a good starting point https://github.com/divio/django-cms/blob/develop/cms/tests/page.py#L1226-L1234

Corrupting the parent node by setting ``parent.get_public_object().get_root().pk`` to something else should be enough to trigger the warning